libs/designsystem/item/src/_item.utils.scss

```diff
@@ -33,4 +33,8 @@
   @include utils.slotted('[detail]') {
     color: utils.get-text-color('semi-dark');
   }
+
+  @include utils.slotted('[wrap]') {
+    white-space: normal !important;
```

Comment:
Is this needed?
```suggestion
    white-space: normal;
```

Reply:
Same for this one. It's actually not necessary